### DISCUSSION TITLE
Testing Plant Health

### DISCUSSION THREAD

**Discussion by christinaballagh** [STELLA-Landsat/STELLA#36] [For Educators]
Try to find the optimal level of watering for a plant by using different amounts of water or time intervals of being watered using the STELLA, and do calculations such as the NDVI or other vegetation indices using the reflectance. A similar experiment could be done to find the best type of soil for that particular plant by comparing different soils.

**Comment by pat-reynolds-81** [comment]
I was looking at something like this, too and am just starting to get my head around NDVI.  I know that NDVI is NIR-Red / NIR+Red, but what values from the STELLA 1.1 device would be used?

From the screen shot, I get that the 'red' would be the R650 value (63.1), but what wavelength would you use for the NIR?

<img width="621" alt="STELLA-SCREEN" src="https://github.com/STELLA-Landsat/STELLA/assets/142839121/fe9f9e4e-3ebd-44d8-ab44-6f36318bbfbe">

**Reply by christinaballagh** [reply]
For the NIR, I would say use the 860nm band, as used by Landsat, and for the red you could do 650, or even 680. Really for both of them you could play around with different red and NIR values to see how it affects the NDVI result. Additionally, it is important to calculate the reflectance, because you use the reflectance values calculated from the irradiance values recorded on the STELLAs for this calculation. Reflectance is the percentage of how much total light is being reflected from the surface of what is being measured, and requires a white calibration medium, such as polystyrene foam. The full calculation for reflectance can be found here: [https://github.com/STELLA-Landsat/STELLA/discussions/14](url).

**Reply by pat-reynolds-81** [reply]
This is so helpful.  Thank you very much!
